src/OpenTelemetry/Trace/Configuration/OpenTelemetrySDK.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="OpenTelemetrySDK.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
```

Comment:
```suggestion
// Copyright 2020, OpenTelemetry Authors
```

Reply:
Wonder if we could remove the specific year like Python is doing: https://github.com/open-telemetry/opentelemetry-python/blob/090b6640495bcb994a9467ab23e2b1abe7fc4af9/opentelemetry-api/src/opentelemetry/trace/__init__.py#L1

Reply:
yep